lib/slack-ruby-bot-server/ext/slack-ruby-bot/commands/base.rb

```diff
@@ -4,16 +4,31 @@ class Base
       class << self
         alias _invoke invoke
 
-        def invoke(client, data)
-          _invoke client, data
-        rescue Mongoid::Errors::Validations => e
-          logger.info "#{name.demodulize.upcase}: #{client.owner}, error - #{e.document.class}, #{e.document.errors.to_hash}"
-          client.say(channel: data.channel, text: e.document.errors.first[1], gif: 'error')
-          true
-        rescue StandardError => e
+        if defined?(::Mongoid)
+          def invoke(client, data)
+            _invoke client, data
+          rescue Mongoid::Errors::Validations => e
```

Comment:
We really should refactor this out of here, but I am not sure how or where. I am OK with this change for now, just thinking out loud.